builder.go

```diff
@@ -248,12 +249,12 @@ func (bom *BOMEntry) convertVersionToMetadata() error {
 	return nil
 }
 
-func setupEnv(env BuildEnv, layersDir string) error {
+func (b *Builder) setupEnv(bpAPI *api.Version, layersDir string) error {
```

Comment:
I don't know what's the reason that Emily made this change, but I think it should be like that because this function relates to the `Builder`.

Reply:
Two reasons:
1. What @yaelharel said. This is specific to the build environment so it makes semantic sense to me to put it Builder rather than having it at the package level.
2. Trying to keep the numbers of parameters manageable. If `*Builder` is the receiver we don't need to pass in the `BuildEnv`

If we have to undo this shortly to merge it with your refactor in #461 that is understandable.

Reply:
Ah yeah, `setupEnv` ended up in `BuildpackTOML.Build()` in my refactor so that prompted the question. But it won't be a big deal to change it later.